docs/modules/reference/pages/glossary.adoc

```diff
@@ -21,15 +20,16 @@ Auto-discovered nodes:: Nodes that are automatically added to {page-component-ti
 (See xref:operation:provisioning/auto-discovery.adoc[Auto-Discovery].)
```

Comment:
Given earlier discussion, we could remove this link and make the heading the link.